pkg/clients/aws.go

```diff
@@ -632,3 +631,20 @@ func DiffLabels(local, remote map[string]string) (addOrModify map[string]string,
 	}
 	return
 }
+
+// CleanError Will remove the requestID from a awserr.Error and return a new awserr.
+// If not awserr it will return the original error
+func CleanError(err error) error {
+	if awsErr, ok := err.(awserr.Error); ok {
+		return awserr.New(awsErr.Code(), awsErr.Message(), nil)
+	}
+	return err
+}
+
+// Wrap Attempts to remove requestID from awserr before calling Wrap
+func Wrap(err error, msg string) error {
+	if err != nil {
+		return errors.Wrap(CleanError(err), msg)
+	}
+	return errors.Wrap(err, msg)
```

Comment:
```suggestion
	return errors.Wrap(CleanError(err), msg)
```

`errors.Wrap` is no-op if error is `nil`

Reply:
went ahead and added